services/db/AxiosInstance.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosRequestConfig, AxiosResponse } from "axios"
+
+import logger from "@logger/logger"
+
+const axios = require("axios")
+
+// Env vars
+const { GITHUB_ORG_NAME } = process.env
+
+const requestFormatter = (config: AxiosRequestConfig) => {
+  logger.info("Making GitHub API call")
+  return {
+    ...config,
+    headers: {
+      ...config.headers,
+      "Content-Type": "application/json",
```

Comment:
would this cause errors in teh future if we have different content types? eg: when we upload a file, the MIME type is no longer json but image. 

if `config.headers["Content-Type"]` is conditionally set, we can instead reverse the order of spreading (we specify content type first then spread headers) to avoid overriding.

Reply:
That's a good catch - it'll be useful to be able to override this in future, although we currently appear to be uploading even images with `application/json`. https://github.com/isomerpages/isomercms-backend/pull/440/commits/36a138e63138508f6ebef4427a9577409263ae12